docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Migrate from Fluffy to lakeFS Enterprise
+description: How to Migrate from a Fluffy-Based Enterprise Version to New lakeFS Enterprise
+---
+
+# Migrate From Fluffy to lakeFS Enterprise
```

Comment:
Migration should always contain very clear steps to execute and checks to perform. 
Im user I want to get from point A to B. How? 
Even if it's obvious, step 1, step 2, etc... + always mentioned if there's downtime involved